libs/credentials/src/validators/easAttestations/index.ts

```diff
@@ -1,10 +1,11 @@
-import { Context, EASContext, Validator } from "../.."
+import { Context, Validator } from "../.."
+import provider from "../../providers/eas"
 
 export type Criteria = {
     minAttestations: number
-    recipient: string
+    schemaId: string
```

Comment:
```suggestion
    schemaId?: string
```
This should be optional too.